pyseir/inference/infer_rt.py

```diff
@@ -394,13 +397,13 @@ def get_posteriors(self, timeseries_type, plot=False):
                 # a single (smoothed) zero value.
                 #
                 # We understand that restarting the posteriors with the
-                # initial prior may incur a start-up artifact as the posterior
+                # re-initial prior may incur a start-up artifact as the posterior
                 # restabilizes, but we believe it's the current best
                 # solution for municipalities that have smoothed cases and
                 # deaths that dip down to zero, but then start to increase
                 # again.
 
-                posteriors[current_day] = prior0
+                posteriors[current_day] = reinit_prior
```

Comment:
This looks good to me. Another option would be, like you say, to just carry forward the last valid day, e.g. ```posteriors[current_day] = posteriors[previous_day]```
I like this solution more, though, as it gives more flexibility for adjusting the re-initialization condition going forward.